go/vt/vtgate/planbuilder/route.go

```diff
@@ -842,18 +844,10 @@ func (rb *route) exprIsValue(expr sqlparser.Expr) bool {
 
 // queryTimeout returns DirectiveQueryTimeout value if set, otherwise returns 0.
 func queryTimeout(d sqlparser.CommentDirectives) int {
-	if d == nil {
-		return 0
-	}
-
-	val, ok := d[sqlparser.DirectiveQueryTimeout]
-	if !ok {
-		return 0
-	}
-
-	intVal, ok := val.(int)
-	if ok {
-		return intVal
+	if val, ok := d[sqlparser.DirectiveQueryTimeout]; ok {
```

Comment:
before we were checking if the incoming map was nil, we don't need to do that anymore?

Reply:
A read from a `nil` map in Go is the same as a read from an empty map! It returns the default value + `false`. So we're handling that just fine!